frontend/src/components/scrimmageRequestForm.js

```diff
@@ -31,12 +31,22 @@ class ScrimmageRequestForm extends Component {
     this.requestScrimmage = this.requestScrimmage.bind(this);
   }
 
+  getRandomMaps(available_maps) {
+    const possible_maps = available_maps.slice();
```

Comment:
you're sure this deep-copies right? potential way to test would be to run this multiple times

Reply:
We don't need a deep copy since we aren't modifying the inner structs, just the array itself